apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -129,6 +155,76 @@ class PassengerForm extends React.Component<Props, State> {
     });
   };
 
+  validateYear = (year: number): boolean => {
+    return year > getYear(new Date()) - 110 && year <= getYear(new Date());
```

Comment:
What about use some const for 110? It could be in this file, but this hardcoded value is hard to read.

Reply:
Fixed, called it maxAge